src/Salt/SiteBundle/Controller/CommentsController.php

```diff
@@ -166,4 +167,59 @@ private function apiResponse($data, $statusCode = 200): JsonResponse
 
         return JsonResponse::fromJsonString($json, $statusCode);
     }
+
+    /**
+     * @Route("/salt/case/export_comment/{itemType}/{id}/comment.csv", name="export_comment_file")
+     *
+     * @param int $id
+     * @param string $itemType
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function exportCommentAction(string $itemType, int $id, Request $request)
+    {
+        $url=$request->getBaseUrl('_route').'/cftree/'.$itemType.'/'.$id;
+        $repo = $this->getDoctrine()->getManager()->getRepository(Comment::class);
+        $rows=array();
+        if($itemType=='item')
+        {
+            $comments=array('Framework Name', 'Node Address', 'HumanCodingScheme', 'User', 'Organization', 'Comment');
+            $rows[]=implode(',', $comments);
+            $comment_data=$repo->findBy(['item'=>$id]);
```

Comment:
You can pass `$itemType` to get the comments data and remove the `if` block condition